### PR TITLE
[DataProtection] Rename the namespace to Azure.ResourceManager.DataProtectionBackup and prepare release

### PR DESCRIPTION
**Pull request by Yao725**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/dataprotection/Azure.ResourceManager.DataProtection/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2022-10-10)
+
+### Breaking Changes
+
+New design of track 2 initial commit.
+
+### Package Name
+
+The package name has been changed from `Microsoft.Azure.Management.DataProtection.Backup` to `Azure.ResourceManager.DataProtection`.
```
Comment (Yao725) [inline sdk/dataprotection/Azure.ResourceManager.DataProtection/CHANGELOG.md:11]: Compared to other languages, it looks like the track 1 package name has an extra suffix `Backup`. For instance, this RP in python is named [azure-mgmt-dataprotection](https://pypi.org/project/azure-mgmt-dataprotection/1.0.0b1/).

Have reached to the service team to double confirm the package name.

**Review by fengzhou-msft — APPROVED** [review]

File: sdk/dataprotection/Azure.ResourceManager.DataProtection/CHANGELOG.md (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/dataprotection/Azure.ResourceManager.DataProtection/CHANGELOG.md:11]: What's the name for Java?

Comment (Yao725) [reply sdk/dataprotection/Azure.ResourceManager.DataProtection/CHANGELOG.md:11]: It is [azure-resourcemanager-dataprotection](https://search.maven.org/artifact/com.azure.resourcemanager/azure-resourcemanager-dataprotection/1.0.0-beta.1/jar).

Comment (Yao725) [reply sdk/dataprotection/Azure.ResourceManager.DataProtection/CHANGELOG.md]: Confirmed with service team, backup is a correct suffix, so use `Azure.ResourceManager.DataProtectionBackup` as the namespace to keep consistent with track 1.

**Pull request merged**